clients/src/main/java/org/apache/kafka/clients/consumer/KafkaShareConsumer.java

```diff
@@ -242,6 +261,71 @@
  * non-transactional records and committed transactional records only. The consumption is bounded by the last stable
  * offset, so an open transaction blocks the progress of the share group with read_committed isolation level.
  *
+ * <h3><a name="renewal">Extending Processing Times using Renewal Acknowledgement</a></h3>
+ * For situations in which the processing time is longer than the acquisition lock duration, the application
```

Comment:
Should we also mention that for this approach the user need to set `share.acknowledgement.mode` to `explicit`?